composer.json

```diff
@@ -63,57 +63,58 @@
     "lint": "phpcs",
     "lint:all": [
       "@lint",
+      "@lint:plugins"
+    ],
+    "lint:plugins": [
```

Comment:
Is there a reason that we need a `lint:plugins` group at all,  instead of the whole list being consolidated under `lint:all`?

Reply:
Actually, no. Fixed it in https://github.com/WordPress/performance/pull/1182/commits/0596b062c52c7f4c584e69c7af3ea63981b274d4